screens/AddPortScreen.tsx

```diff
@@ -1,10 +1,23 @@
-import { View, Text } from 'react-native'
+import { View, Text, SafeAreaView } from 'react-native'
+import { Button, Card, Chip, Input } from "@rneui/themed";
 import React from 'react'
+import { InputForm } from '../components/ui/InputForm';
 
-export default function AddPortScreen() {
+export default function AddPortScreen({ navigation }) {
   return (
-    <View>
-      <Text>AddPortScreen</Text>
-    </View>
+    <SafeAreaView style={{ backgroundColor: "#FFFFFF" }}>
+      <Card containerStyle={{ borderRadius: 20, padding: 25 }}>
+      <InputForm title='ชื่อโครงงานภาษาไทย' />
```

Comment:
อย่าใช้ inputForm ให้เรียกใช้ Input ตรงๆเลย เพราะจะทำ backend ยาก
```suggestion
      <Text style={styles.formText}>{'ชื่อโครงงานภาษาไทย'}</Text>
      <Input  style={styles.textInput}  inputContainerStyle={{ borderBottomWidth: 0 }}/>
```